pkgs/applications/emulators/retroarch/retroarch-assets.nix

```
@@ -6,7 +6,7 @@
 
 stdenvNoCC.mkDerivation {
   pname = "retroarch-assets";
-  version = "unstable-2024-04-24";
+  version = "0-unstable-2024-04-24";
```

```suggestion
  version = "1.17.0-unstable-2024-04-24";
```

seems like this has a proper tag?
https://github.com/libretro/retroarch-assets/releases/tag/v1.17.0

---

Oops, yup. Fixed.